logging/v5_7/logging-5-7-architecture.adoc

```diff
@@ -8,7 +8,4 @@ toc::[]
 
 :context: logging-5-7-architecture-overview
 include::modules/logging-architecture-overview.adoc[lines=9..31]
-
-include::modules/logging-support-considerations.adoc[leveloffset=+1]
-
 include::snippets/logging-5.7-outputs-snip.adoc[]
```

Comment:
This isn't displaying correctly in the preview file.

Reply:
Fixed